support-frontend/assets/helpers/globals.js

```diff
@@ -16,46 +16,33 @@ function getGlobal<T>(path: string = ''): ?T {
   return null;
 }
 
+const emptyAmountsSettings = {
+  ONE_OFF: [],
+  MONTHLY: [],
+  ANNUAL: [],
+};
+
 const getSettings = (): Settings => getGlobal('settings') || {
   switches: {
     experiments: {},
   },
   amounts: {
-    GBPCountries: {
-      ONE_OFF: [],
-      MONTHLY: [],
-      ANNUAL: [],
-    },
-    UnitedStates: {
-      ONE_OFF: [],
-      MONTHLY: [],
-      ANNUAL: [],
-    },
-    EURCountries: {
-      ONE_OFF: [],
-      MONTHLY: [],
-      ANNUAL: [],
-    },
-    AUDCountries: {
-      ONE_OFF: [],
-      MONTHLY: [],
-      ANNUAL: [],
-    },
-    International: {
-      ONE_OFF: [],
-      MONTHLY: [],
-      ANNUAL: [],
-    },
-    NZDCountries: {
-      ONE_OFF: [],
-      MONTHLY: [],
-      ANNUAL: [],
-    },
-    Canada: {
-      ONE_OFF: [],
-      MONTHLY: [],
-      ANNUAL: [],
-    },
+    GBPCountries: emptyAmountsSettings,
+    UnitedStates: emptyAmountsSettings,
+    EURCountries: emptyAmountsSettings,
+    AUDCountries: emptyAmountsSettings,
+    International: emptyAmountsSettings,
+    NZDCountries: emptyAmountsSettings,
+    Canada: emptyAmountsSettings,
+  },
+  contributionTypes: {
+    GBPCountries: [],
```

Comment:
I wonder if it's worth combining the amounts + contributionTypes settings into one model, e.g.

```
type Amount = {
  value: string,
  isDefault?: boolean,
}

type ContributionTypeSetting = {
  contributionType: ContributionType,
  isDefault?: boolean,
  amounts: Amounts[],
}

type ContributionTypesSettings = {
  [CountryGroupId]: ContributionTypeSetting[],
}
```

Reply:
Yes! We could put everything that's specific to region + contribution type in there.

Reply:
(e.g. possibly the "editorialise amounts" copy, `Contributing £50 works out as 14p a day`)

Reply:
the migration could be quite complicated so I'll create a trello card to address this